pkg/scheduler/framework/plugins/podtopologyspread/filtering.go

```diff
@@ -331,7 +331,7 @@ func (pl *PodTopologySpread) Filter(ctx context.Context, cycleState *framework.C
 		tpKey := c.TopologyKey
 		tpVal, ok := node.Labels[c.TopologyKey]
```

Comment:
Could you change to use tpKey here:
```suggestion
		tpVal, ok := node.Labels[tpKey]
```